ground/src/main/java/com/google/android/ground/ui/map/gms/mog/MogCollection.kt

```diff
@@ -17,21 +17,22 @@
 package com.google.android.ground.ui.map.gms.mog
 
 /** A collection of Maps Optimized GeoTIFFs (MOGs). */
-@Suppress("MemberVisibilityCanBePrivate")
 class MogCollection(
-  val worldMogUrl: String,
-  val hiResMogUrl: String,
+  private val worldMogUrl: String,
```

Comment:
Note that these were public because the idea was that this can be a reusable package, but since it's not atm I'm fine with keeping these as private.